CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Setting ownership to the DevX & Modules teams
+* @puppetlabs/devx
+* @puppetlabs/modules
```

Comment:
I think these need to be on the same line. As I read the spec, this makes @puppetlabs/modules owners of everything, not devx

Reply:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

Reply:
ope, gotcha